### ISSUE TITLE
Android ToggleButtons are Missing Borders

### ISSUE THREAD

**Issue by robloo** [unoplatform/uno#7192]
### Current behavior

The latest release of Uno 3.10.7 broke borders around ToggleButton.

![image](https://user-images.githubusercontent.com/17993847/134055206-2dd75247-f3df-4930-b38f-de2c0f5e46a5.png)

This may be related to the Fluent version 2 theme changes for borders, or BackgroundSizing support https://github.com/unoplatform/uno/issues/6637#issuecomment-923176257

### Expected behavior

![image](https://user-images.githubusercontent.com/17993847/127912702-eda33f7a-bbeb-4d04-9ab6-bf69d2cef605.png)

(ignore the theme)

### Workaround

None

### Works on UWP/WinUI

Yes

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### NuGet package version(s)

3.10.7

### Affected platforms

Android

### IDE

Visual Studio 2019

**Comment by jeromelaban** [comment]
Thanks for the report.

I'm not getting this result with 3.10.7 (or 3.11 dev):

![image](https://user-images.githubusercontent.com/5839577/134578997-4b6abdec-fd63-4aa3-86b5-1e7e5bd52b03.png)

Are you using the same XAML as the previous issue ?

**Comment by robloo** [comment]
@jeromelaban 

> I'm not getting this result with 3.10.7 (or 3.11 dev):

Interesting, I'm seeing this both on an emulator and a physical device. Emulator is Pixel 2 Q 10.0 API 29.

> Are you using the same XAML as the previous issue ?

Yes, the same XAML. However, this is seen in other areas as well. All places using this type of XAML see the issue. I should note that this only happens on ToggleButtons with a corner radius.

![asdfasdf](https://user-images.githubusercontent.com/17993847/134580308-4c3d859b-a103-4e6f-a40a-a2754991accb.gif)

**Comment by jeromelaban** [comment]
In this case, can you create an exact repro of the app where you see the issue? There may be an additional configuration that is different.

**Comment by robloo** [comment]
> In this case, can you create an exact repro of the app where you see the issue? There may be an additional configuration that is different.

Yes, I should add WinUI library is being used along with Fluent v1.

**Comment by robloo** [comment]
@jeromelaban 

I confirmed the issue is due to Fluent version 1 styles. Something broke there with the latest updates. Repro is attached to the issue description above.

![image](https://user-images.githubusercontent.com/17993847/134586790-ce50d343-63e6-4b95-b31b-a327ed5bcc88.png)

**Comment by jeromelaban** [comment]
Thanks. I'm not seeing a repro solution zipped in the description. Are you going to attach it?

**Comment by robloo** [comment]
@jeromelaban Just added it. I ended up having to clean a few things up and it took a bit longer than expected (deleting bin/obj and unused platform heads). If it doesn't load I deleted a line I shouldn't have in the solution. I expect it would work though.

**Comment by jeromelaban** [comment]
Thanks. It's indeed the V1 of styles that causes this issue. It's related to the support for `BackgroundSizing` which is not fully implemented in Uno Android/iOS and causes  rendering issues when the corner radius is set. (/cc @carldebilly)

I would suggest that you change your styles to use a copy the V2 style for that control, which uses a different layout for rendering the border:

https://github.com/unoplatform/uno/blob/fe053311cf33416722e7e1ffeab047c36a142fd3/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/ToggleButton_themeresources.xaml#L193


![image](https://user-images.githubusercontent.com/5839577/134607005-9e49f3f8-6bbb-4662-a661-6aff847ef593.png)

**Comment by robloo** [comment]
It's on the roadmap to update to fluent v2 now before the next release. So that would include the work-around for this. However, it probably should also work on Fluent v1 out of the box.

**Comment by jeromelaban** [comment]
@carldebilly can you rename this issue to be aligned with the `BackgroundSizing` feature implementation ?

**Comment by MartinZikmund** [comment]
It seems `BackgroundSizing` is not yet supported on Skia? Needs to be validated and added if not supported yet

**Comment by robloo** [comment]
Here is the MIT licensed Avalonia implementation if it helps: https://github.com/AvaloniaUI/Avalonia/pull/14048

This was also based on WinUI with some additions.

**Comment by MartinZikmund** [comment]
# Repro generator

This issue was classified as not reproducible with a code sample or test.

**Reason:** Feature request for BackgroundSizing support — styled as bug but describes missing API.
